docs/src/Tari_FAQ.md

```diff
@@ -0,0 +1,252 @@
+
+### What is the coin name? And what is it abbreviation and symbol?
+Minotari. The abbreviation and symbol is XTM.
+
+### Is Tari open-source?
+Yes.
+
+### What is the official Github repository for the Tari Project?
+Tari Project's repos can be found in [https://github.com/tari-project/](https://github.com/tari-project/). The main project is [https://github.com/tari-project/tari/](https://github.com/tari-project/)
+
+### Where can I find installation instructions for Tari?
+Normal users can utilise the binaries found on Tari's main website under [www.tari.com/downloads](https://tari.com/downloads/). Developers should refer to the instructions available on in the main Tari repo at [https://github.com/tari-project/tari/](https://github.com/tari-project/)
+
+### Explain the general architecture of the Tari solution? What components make up Tari?
+Tari consists of several components. While it is not necessary to run all of these as a casual user, all are required if you wish to run a full base node with mining capabilities. The components are:
+
+- Minotari Base Node (`minotari_node`): Validates and propagates Tari coin transactions and blocks on the base layer. Forms the backbone of the Tari blockchain, maintaining consensus about the longest valid proof-of-work blockchain
+- Minotari Console Wallet (`minotari_console_wallet`):  Manages user funds, creates transactions, and maintains private keys. Allows users to send, receive, and manage their Tari coins. The wallet also provides a means to interact with the gRPC Wallet API. A mobile wallet is also available via [Aurora](https://aurora.tari.com)
+- Minotari Miner (`minotari_miner`): Enables merge mining with Monero. Connects XMRig (Monero mining software) to the Tari network to allow simultaneous mining of both Minotari and Monero. This will be dependent on XMRig and the `minotari_merge_mining_proxy`.
+- Tari Universe: A desktop-based client that handles the installation and configuration of Tari in a simple installation. Recommended for the average user.
+
+Additional, dependent components that aren't specific to Tari are:
+
+- XMRig: Third-party Monero mining software. Used in conjunction with the `minotari_merge_mining_proxy` to mine both Monero and Tari.
+- Tor: Provides network privacy and NAT traversal. Enables secure, anonymous communication between Tari nodes.
+
+### Are there any Tari blockchain explorers?
+The mainnet explorer will be [https://explore.tari.com](https://explore.tari.com).
+
+Please note that only Testnet explorer is currently live: https://explore-nextnet.tari.com/
+
+## Tokenomics
+### Is there any documentation available describing the tokenomics of Tari?
+Users can refer to the following: [https://tari.substack.com/p/tari-tokenomics](https://tari.substack.com/p/tari-tokenomics)
+
+### Is there an initial total supply? And is the supply fixed?
+Excluding premine: 0 at genesis, 21 billion XTM once tail emission starts. 
```

Comment:
you say 0 here, few lines later you correctly state `Protocol Infrastructure and Grants: 40% (756 million tokens) available at launch for protocol liquidity, `